third_party/blink/renderer/core/events/message_event.cc

```diff
@@ -222,6 +222,10 @@ MessageEvent* MessageEvent::Create(const AtomicString& type,
         "The optional 'source' property is neither a Window nor MessagePort.");
     return nullptr;
   }
+
+  // https://linear.app/replay/issue/RUN-1150
+  recordreplay::Assert("[RUN-1150] MessageEvent::Create #1");
```

Comment:
* Q1: Would it make sense to add the `type` into the assertion?
* Q2: How did you find that this was related from the `crashReport`?

Reply:
Q1: We're just making sure there is an assertion before the point when the event constructor reads the current time, so the type string isn't relevant.
Q2: This is actually from https://linear.app/replay/issue/RUN-1146 which was duped to RUN-1150, as the call stacks between the two issues are very similar.